graphrag/logging/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""Logging utilities and implementations."""
+
+from .console import ConsoleReporter  # noqa: I001
```

Comment:
Why don't we sort the imports here?

Reply:
Thanks for catching this. I previously encountered an error with circular imports that required these imports to be un-ordered but that appears to no longer be an issue. Once the factory design is fully implemented for certain top-level modules, the risk of circular imports will be even less.